examples/tree_2d_dgsem/elixir_navier_stokes_convergence.jl

```diff
@@ -25,152 +23,8 @@ mesh = TreeMesh(coordinates_min, coordinates_max,
                 periodicity=(true, false),
                 n_cells_max=30_000) # set maximum capacity of tree data structure
 
-# Define initial condition
-# Note: If you change the parameters here, also change it in the corresponding source terms
-function initial_condition_navier_stokes_convergence_test(x, t, equations)
-
-  # Amplitude and shift
-  A    = 0.5
-  c    = 2.0
-
-  # convenience values for trig. functions
-  pi_x = pi * x[1]
-  pi_y = pi * x[2]
-  pi_t = pi * t
-
-  rho = c + A * sin(pi_x) * cos(pi_y) * cos(pi_t)
-  v1  = sin(pi_x) * log(x[2] + 2.0) * (1.0 - exp(-A * (x[2] - 1.0)) ) * cos(pi_t)
-  v2  = v1
-  p   = rho^2
-
-  return prim2cons(SVector(rho, v1, v2, p), equations)
-end
-
 initial_condition = initial_condition_navier_stokes_convergence_test
 
-@inline function source_terms_navier_stokes_convergence_test(u, x, t, equations)
-
-  y = x[2]
-
-  # we currently need to hardcode these parameters until we fix the "combined equation" issue
-  # see also https://github.com/trixi-framework/Trixi.jl/pull/1160
-  kappa = 1
-  inv_gamma_minus_one = inv(equations.gamma - 1)
-  Pr = get_Pr()
-  Re = get_Re()
```

Comment:
The problem with moving these to `src` is that we need `reynolds_number()` etc. defined above in this file.

Reply:
Ah...yes, that's true. We can still run the elixir so long as those functions are defined, but it does get messy. 

@sloede ok to move the manufactured CNS solution back into the elixirs for now?

Reply:
Argh... One more reason to reconsider our data structures 😬 Sure, just leave it as it is then.